pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -78,7 +78,7 @@ var _ = Describe("Node-labeller config", func() {
 		cpuModels := nlController.getSupportedCpuModels(nlController.clusterConfig.GetObsoleteCPUModels())
 		cpuFeatures := nlController.getSupportedCpuFeatures()
 
-		Expect(cpuModels).To(HaveLen(5), "number of models must match")
+		Expect(cpuModels).To(HaveLen(4), "number of models must match")
```

Comment:
We should really switch to `HaveElements(...)`

Reply:
I fully agree, but it is out of the scope of this PR.